packages/@teip/create-flow-types/src/util.js

```diff
@@ -88,5 +90,27 @@ export function mergeObjectTypes(
       propertyMap.set(property.key.name, property);
     }
   });
-  return T.objectTypeAnnotation([...propertyMap.values()]);
+  //const unnestedTypes = unnestUnionTypes(types);
+  //const unnestedTypes = types;
```

Comment:
I think these comments are left over from some kind of refactor?

Reply:
Yes thanks :).